index.html

```diff
@@ -0,0 +1,451 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="grid">
+      <section class="home">
+        <header>
```

Comment:
- [ ] It is like you broke the mobile version UI while coding up the desktop version. 
- The hamburger menu disappeared
- There is, now, empty space to both ends of the mobile version
- The font size of the name at the top-left decreased
Please, :pray: fix this and your app will definitely look better.
## Your mobile version presently
![image](https://user-images.githubusercontent.com/45668340/145482468-d352b202-2489-4a1d-ad4e-304cf5db5f91.png)

## Design template
![image](https://user-images.githubusercontent.com/45668340/145482238-51083b46-5ca7-48a1-8690-c50a3ace9ef8.png)

Reply:
Well, the fact is that not the hamburger disappeared, but that according to Figma's design in the desktop version it is not the hamburger, but the menu that I used, if I install a hamburger, it will be a violation of Figma's design. 
![home](https://user-images.githubusercontent.com/52704856/145532122-26c0cc99-8360-4dd4-bbf5-9d7ebab777b1.png)

Please, answer me so that I can continue working on your suggestions.

Reply:
Thank you, @Gegardus, for your question. You can get the inter font in google font.
Check it out here => [Inter font](https://fonts.google.com/specimen/Inter)

Reply:
I actually checked your mobile version, not the desktop in the **hamburger** case. I am supposed to make sure that while you work on the desktop version, your mobile version is still intact. You can do this, I am sure because you have done it before.
## Happy coding:tada:

Reply:
Well, I am in a complete mess for now because I can't understand, should I implement Figma's desktop design or mobile design? Or I should have two different designs, and when it goes after 1100 width one design should change the previous design?

Reply:
Please, contact me on slack. Let's have a zoom call. It's all about media query I think.

Reply:
The two designs stays. One design replaces the other depending on the screen size just like you said.

Reply:
Checkout this project for example. 


https://henrykc24.github.io/my-portfolio/

Reply:
Thank you, Henry I really misunderstand the project's main requirement. Now I should try to improve it on time.

Reply:
Hi @Gegardus, I perfectly understand how difficult it can be at first at Microverse because I was there too. One step at a time and I am sure you are going to pull through just like I did. Feel free to reach out anytime you need some help. 
## Happy coding:tada: